### PR TITLE
Fixing small typo in search_dsl.rst

### PR DESCRIPTION
**Pull request by mmakridi**
Added missing single quotation mark in line 454 (example snippet for MoreLikeThis query)

### CODE REVIEW

**Review by sethmlarson — APPROVED** [review]
Thanks for this!

**Comment by mmakridi** [comment]
@sethmlarson I have merged the master branch to resolve the auto-merge conflict. However, Github now shows me "1 workflow awaiting approval" message. Is there anything I can do to resolve it on my side, or is it really something that a maintainer should perform for the workflows to proceed?

**Pull request merged**
